package.json

```diff
@@ -68,8 +68,13 @@
     "typescript": "^5.0.2"
   },
   "dependencies": {
+    "@scure/base": "^1.1.6",
+    "@scure/bip32": "^1.4.0",
+    "@scure/bip39": "^1.3.0",
     "@bsv/sdk": "^1.0.33",
     "@types/rollup-plugin-node-builtins": "^2.1.5",
-    "cross-fetch": "^4.0.0"
+    "cross-fetch": "^4.0.0",
+    "hi-base32": "^0.5.1",
```

Comment:
We should check whether or not we want to rely on this lib. (its only single file src) https://github.com/emn178/hi-base32/blob/master/src/base32.js

Maybe we could embed our own implementation - it doesn't seem challenging

Reply:
Removed this package